src/js/pannellum.js

```diff
@@ -1657,7 +1657,7 @@ function render() {
             compass.style.transform = 'rotate(' + (-config.yaw - config.northOffset) + 'deg)';
             compass.style.webkitTransform = 'rotate(' + (-config.yaw - config.northOffset) + 'deg)';
         }
-        fireEvent('renderFinished', {pitch: _this.getPitch(), yaw: _this.getYaw(), hfov: _this.getHfov()})
+        fireEvent('renderFinished', {sceneId: config.scene, pitch: config.pitch, yaw: config.yaw, hfov: config.hfov});
```

Comment:
getYaw isn't just a getter, it does some clamping (https://github.com/disperse-io/pannellum/blob/f9a5c6a26f0973fa522beb280efe8df76a145cd1/src/js/pannellum.js#L2765).

The others are fine, but as the code elsewhere uses those getters internally I'd be inclined towards keeping the approach consistent, ymmv.